CODEOWNERS

```diff
@@ -11,21 +11,17 @@ go.mod @fleetdm/go
 /infrastructure/ @zwinnerman-fleetdm @edwardsb @rfairburn
 /charts/ @zwinnerman-fleetdm @edwardsb @rfairburn
 
-# GitHub settings + actions
-/.github/ @fleetdm/g-platform
+# GitHub settings + actions + issue templates
+/.github/ @zwass @mikermcneil
```

Comment:
since this github user group didn't exist anymore, this was causing a "this CODEOWNERS file is invalid" error.

Reply:
got it, thanks for the clarification.